.dev/cc.R

```diff
@@ -81,7 +81,7 @@ cc = function(test=FALSE, clean=FALSE, debug=FALSE, omp=!debug, cc_dir, path=Sys
     .GlobalEnv[[Call$name]] = Call$address
   for (Extern in xx$.External)
     .GlobalEnv[[Extern$name]] = Extern$address
-  sourceDir(file.path(path, "R"))
+  sourceDir(file.path(path, "R"), trace=!quiet)
   if (base::getRversion()<"4.0.0") rm(list=c("rbind.data.table", "cbind.data.table"), envir=.GlobalEnv) # 3968 follow up
```

Comment:
(for potential follow-up)

is it still useful to have such a condition in a script for dev work? should any dev work still be happening with now-several-years-old R? If this is for back-compatibility testing, wouldn't we want some flexibility in the `R CMD` calls invoked by `cc()` to instead be some other R binary?